utils/gyb_syntax_support/protocolsMap.py

```diff
@@ -4,12 +4,20 @@
         'MemberDeclListItem',
         'SyntaxBuildable'
     ],
-    'StmtBuildable': [
```

Comment:
Made then in alphabetical order so it's easier to find them. 
Would you mind to verify that?